### PR TITLE
Add a node about physical behaviors of non top robot nodes

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Try to document why a Robot inserted in another Robot node doesn't behave the same way as if it would be a top node.

This is maybe not necessary because it follows the same rules as for any Solid node.
But given that robots could move I think it is not bad to explicitly write it again.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/robot.md [hunk 1]
```diff
@@ -21,9 +21,11 @@ Direct derived nodes: [DifferentialWheels](differentialwheels.md), [Supervisor](
 
 ### Description
 
-The [Robot](#robot) node can be used as basis for building a robot, e.g., an articulated robot, a humanoid robot, a wheeled robot...
+The [Robot](#robot) node can be used as basis for building a robot, e.g., an articulated robot, a humanoid robot, a wheeled robot.
 If you would like to build a robot with supervisor capabilities use the [Supervisor](supervisor.md) node instead (Webots PRO license required).
 
+> **Note**: If the Robot node has one or more Solid (or derived) ancestor nodes, then the physical properties of the ancestor nodes will affect the Robot node physical behavior.
```
Comment (omichel) [inline reference/robot.md]: I would this sentence with "Obviously": "Obviously, if the Robot node..."

Comment (tn12787) [reply reference/robot.md]: "Obviously" is very harsh like this. I would say "Logically".

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**